src/components/storagePools/createStoragePoolDialog.jsx

```diff
@@ -64,6 +64,8 @@ const StoragePoolTypeRow = ({ onValueChanged, dialogValues, libvirtVersion }) =>
     if (libvirtVersion && libvirtVersion >= 4007000)
         poolTypes.push({ type: 'iscsi-direct', detail: _("iSCSI direct target") });
 
+    const supportedPoolTypes = poolTypes.filter(pool => poolCapabilities[pool.type].supported);
```

Comment:
This is where it crashes on debian-stable, `poolCapabilities.dir` is empty.  I suggest this:

    poolCapabilities[pool.type] ? poolCapabilities[pool.type].supported : true

i.e. it falls back to the old behaviour of offering all pool types. On these old distros without runtime detection we'd continue to get the dialog inline error message, which is not a regression (and also the best we can do anyway).